add deprecation notices (#305)
csstools/postcss-plugins: plugins/postcss-env-function/README.md

Currently only valid [custom property names] (beginning with `--`) are accepted.
Not all valid [declaration value names] are accepted.

### disableDeprecationNotice

Silence the deprecation notice that is printed to the console when using `importFrom``.

> > "importFrom" will be removed in a future version of postcss-env-function.

Author: ```suggestion
> postcss-env-function is deprecated and will be removed.
```
